File: Laba5/OP2,5Python/OP2,5Python/square.py
import math
from rectangle import Rectangle
class Square(Rectangle):
    def __init__(self, Coords):
        super().__init__(Coords, "Square")
        self.area = self.countArea()
    def countArea(self):
        Side = math.sqrt(pow(self.Coords[1].x - self.Coords[0].x, 2) + pow(self.Coords[1].y - self.Coords[0].y, 2))
        return pow(Side, 2)

File: Laba5/Laba2.5/Laba2.5/square.cpp
#include "square.h"

double square::countArea() { // Обрахунок площі
	double Side;
	Side = sqrt(pow(V[1].x - V[0].x, 2) + pow(V[1].y - V[0].y, 2)); // Довжина однієї сторони
	return pow(Side, 2);
}
